src/components/LoadingMessage/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import './LoadingMessage.scss';
```

Comment:
So is the CSS compiled at the component level and inserted into the DOM? Although prospectus doesn't do it, I still prefer external stylesheets (and plan on updating prospectus at some point to use that approach).